crates/bevy_pbr/src/render_graph/pbr_pipeline/pbr.frag

```diff
@@ -277,6 +285,71 @@ vec3 reinhard_extended_luminance(vec3 color, float max_white_l) {
 
 #endif
 
+vec3 do_point_light(PointLight light, float roughness, float NdotV, vec3 N, vec3 V, vec3 R, vec3 F0, vec3 diffuseColor) {
```

Comment:
nit: I think `do_x` is equivalent to `x` for functions. The "do" action is implicit when calling a function. I think `point_light` is a solid name as-is.